packages/im-studio/src/organisms/hotspots/Hotpots.js

```diff
@@ -0,0 +1,157 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+// Components
+import { Table } from 'antd';
+import Button from "imcomponents/atoms/button";
+import Form from "imcomponents/atoms/form"
+import Input from "imcomponents/atoms/input";
+import { EditOutlined, DeleteOutlined } from "imcomponents/atoms/icon";
+import TextArea from "imcomponents/atoms/textArea";
+
+// Styles
+import styles from "./hotspots.module.scss";
+
+// Helper
+import { isBackDisabled } from "../../utils/tabHelper.general";
+
+const columns = [
+    {
+        title: 'No.',
+        dataIndex: 'serialNumber',
+        // render: text => <a>{text}</a>
```

Comment:
can u clean all the comments? or maybe add an example with comment so that i would know, whats that use for